modAionImpl/src/org/aion/zero/impl/config/CfgDbDetails.java

```diff
@@ -13,10 +13,10 @@
 public class CfgDbDetails {
 
     public static final int MIN_FD_OPEN_ALLOC = 1024;
-    public static final String DEFAULT_BLOCK_SIZE = "16mB";
-    public static final String DEFAULT_WRITE_BUFFER_SIZE = "64mB";
-    public static final String DEFAULT_READ_BUFFER_SIZE = "64mB";
-    public static final String DEFAULT_CACHE_SIZE = "128mB";
```

Comment:
I would be tempted to remove these configurations and stick to our predefined settings